Source/Core/DolphinQt/Config/VerifyWidget.cpp

```diff
@@ -117,7 +117,7 @@ void VerifyWidget::Verify()
   verifier.Finish();
 
   DiscIO::VolumeVerifier::Result result = verifier.GetResult();
-  progress->setValue(verifier.GetBytesProcessed() / DIVISOR);
+  progress->reset();
```

Comment:
This was a little confusing to me, turns out there's a `autoClose` property which hides the window when `reset()` is called..

This could probably allocated on the stack instead of the heap, especially if this is supposed to be modal/blocking, but LGTM otherwise.

Reply:
I put it in a new PR since this one already is merged: #8306